client/src/elements/molecules/item-list-entry.css

```diff
@@ -32,24 +24,110 @@ item-list-entry > a {
   width: 100%;
 }
 
-item-list-entry a > span {
+item-list-entry a > span:not(.item-list-entry__state-tag) {
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  flex-grow: 1;
```

Comment:
this makes the "columns" grow based on the content. which becomes different as more items get loaded -> layout jumps with infinite scroll.

Reply:
I can not reproduce this issue. The columns take always equal space thats because all columns have `flex-grow` set to 1. If one column has a longer title it will get wrapped as shown in the example below:

<img width="1379" alt="bildschirmfoto 2018-06-21 um 12 26 46" src="https://user-images.githubusercontent.com/1810384/41713812-91383c60-754e-11e8-9b80-85be17385aa4.png">

I think we can keep it like this.